src/math/vec2.js

```diff
@@ -297,6 +386,27 @@ class Vec2 {
         return this;
     }
 
+    /**
+     * @function
+     * @name Vec2#mulScalar
+     * @description Multiplies each element of a vector by a number.
+     * @param {number} scalar - The number to multiply by.
+     * @returns {Vec2} Self for chaining.
+     * @example
+     * var vec = new pc.Vec2(3, 6);

```

Comment:
should these be using let?

Reply:
Yes, we can start thinking about switching to ES6 for JSDoc example code. But not in this PR. 😄